src/logic/embedding.lean

```diff
@@ -232,8 +232,8 @@ open sum
 def sum_map {α β γ δ : Type*} (e₁ : α ↪ β) (e₂ : γ ↪ δ) : α ⊕ γ ↪ β ⊕ δ :=
 ⟨sum.map e₁ e₂,
     assume s₁ s₂ h, match s₁, s₂, h with
-    | inl a₁, inl a₂, h := congr_arg inl $ e₁.injective $ inl.inj h
-    | inr b₁, inr b₂, h := congr_arg inr $ e₂.injective $ inr.inj h
+    | sum.inl a₁, sum.inl a₂, h := congr_arg sum.inl $ e₁.injective $ sum.inl.inj h
+    | sum.inr b₁, sum.inr b₂, h := congr_arg sum.inr $ e₂.injective $ sum.inr.inj h
```

Comment:
Maybe the cause is new imports making `inl` refer to something else? What does `#check inl` produce in this file?

Reply:
It's weird, the problem seems to have gone away for me. I'll try to undo this.